framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -381,9 +378,64 @@ public void processClassTree(ClassTree classTree) {
 
         checkExtendsImplements(classTree);
 
+        checkPolymorphicClass(classTree);
+
         super.visitClass(classTree, null);
     }
 
+    /**
+     * Throws an error if {@code classTree} has polymorphic fields but is not annotated with
```

Comment:
Should be `Issues` an error.   Using `throws` would imply that an exception was thrown.